test/hotspot/jtreg/vmTestbase/nsk/jvmti/RunAgentThread/agentthr001/agentthr001.cpp

```diff
@@ -137,6 +138,7 @@ sys_thread_2(jvmtiEnv* jvmti, JNIEnv* jni, void *p) {
 static void JNICALL
 sys_thread_3(jvmtiEnv* jvmti, JNIEnv* jni, void *p) {
     while (1) {
+      sleep(1);
```

Comment:
Maybe you could add a comment here that the call to sleep is necessary to avoid the compiler optimization to elide the loop.